lib/buffer.js

```diff
@@ -107,7 +111,7 @@ function Buffer(arg, encodingOrOffset, length) {
         'If encoding is specified then the first argument must be a string'
       );
     }
-    return Buffer.allocUnsafe(arg);
+    return allocate(arg);
```

Comment:
@RReverser do you have any benchmarking numbers of the effect of this? If it’s negligible, I’d like to keep it as `Buffer.allocUnsafe`.